src/components/Main.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+import PhotoCard from "./PhotoCard";
+
+import axios from "axios";
+
+function Main() {
+  const [data, setData] = useState({});
```

Comment:
Great insight to set the default state to an empty object, that way we don't get errors when trying to destructure it if it was left undefined.